starter-code/scripts/articleView.js

```diff
@@ -18,29 +18,45 @@ articleView.populateFilters = function() {
 
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-  if ($(this).val()) {
-      /* TODO: If the slect box changes to an option that has a value, we should:
+    if ($(this).val()) {
+      $('article').hide();
+      $(`article[data-author="${ $(this).val() }"]`).fadeIn(500);
+      /* TODO: If the select box changes to an option that has a value, we should:
           1. Hide all of the articles
           2. Fade in only the articles that match based on on the author
-            that was aselected. Hint: use an attribute selector to find
+            that was a-selected. Hint: use an attribute selector to find
             those articles that match the value, and then fade them in.
         */
-  } else {
+    } else {
     /* Otherwise, we should:
         1. Show all the articles except the template */
-  }
+      $('article').not('.template').show();
```

Comment:
Great job utilizing the not method! Again, as I mentioned above, it would be a good practice to be more specific in the articles that you're selecting for.